sljit_src/sljitConfigInternal.h

```diff
@@ -122,6 +105,27 @@ extern "C" {
 #error "Multiple architectures are selected"
 #endif
 
+#if !(defined SLJIT_CONFIG_X86_32 && SLJIT_CONFIG_X86_32) \
+	|| !(defined SLJIT_CONFIG_X86_64 && SLJIT_CONFIG_X86_64) \
```

Comment:
makes more sense with A && B .. && Z or !(A || B .. || Z) but funny enough this also works because of the rule above that prevents having more than one of them